src/content/configuration/output.md

```diff
@@ -618,6 +619,13 @@ define([], function() {
 This bundle will not work as expected, or not work at all (in the case of the almond loader) if loaded directly with a `<script>` tag. It will only work through a RequireJS compatible asynchronous module loader through the actual path to that file, so in this case, the `output.path` and `output.filename` may become important for this particular setup if these are exposed directly on the server.
 
 
+`libraryTarget: "amd-require"` - This packages your output with an immediately-executed AMD `require(dependencies, factory)` wrapper.
+
+The "amd-require" target allows for the use of AMD dependencies without needing a separate later invocation. As with the `"amd"` target, this depends on the appropriate [`require` function](https://github.com/amdjs/amdjs-api/blob/master/require.md) being available in the environment in which the webpack output is loaded.
```

Comment:
i think it should be
```
`"amd-require"`
```

to be consistent with similar syntax in same paragraph where we mention
```
`"amd"`
```